packages/shared/src/utils/currency/currency.ts

```diff
@@ -36,21 +58,27 @@ export const reorderCurrencies = (list, type = 'fiat') => {
 
 export const AMOUNT_MAX_LENGTH = 10;
 
-export const getRoundedNumber = (number, currency) => {
+export const getRoundedNumber = (number: number, currency: string) => {
     return Number(Number(number).toFixed(getDecimalPlaces(currency)));
 };
 
-export const getFormattedText = (number, currency) => {
+export const getFormattedText = (number: number, currency: string) => {
     return `${addComma(number, getDecimalPlaces(currency), isCryptocurrency(currency))} ${currency}`;
 };
 
-export const formatMoney = (currency_value, amount, exclude_currency, decimals = 0, minimumFractionDigits = 0) => {
-    let money = amount;
+export const formatMoney = (
+    currency_value: string,
+    amount: number | string,
+    exclude_currency?: number,
+    decimals = 0,
+    minimumFractionDigits = 0
+) => {
+    let money: number | string = amount;
     if (money) money = String(money).replace(/,/g, '');
     const sign = money && Number(money) < 0 ? '-' : '';
     const decimal_places = decimals || getDecimalPlaces(currency_value);
 
-    money = isNaN(money) ? 0 : Math.abs(money);
+    money = isNaN(+money) ? 0 : Math.abs(+money);
```

Comment:
Maybe it is better to use one style, I mean to use Number() as it is on line 78, and as we use String.
But it is just a suggestion. I'm ok with '+'
as I know, '+' is the fasted method :)

Reply:
We can't just use number type here because we are passing string to this function too